test/test_line.py

```diff
@@ -358,6 +358,44 @@ def test_meth_colliderect(self):
         with self.assertRaises(TypeError):
             A.colliderect(1, 5)
 
+    def test_meth_move(self):
+        line = Line(1.1, 2.2, 3.3, 4.4)
+
+        ret = line.move(1, 2)
+
+        self.assertEqual(ret.x1, 2.1)
+        self.assertEqual(ret.y1, 4.2)
+        self.assertEqual(ret.x2, 4.3)
+        self.assertEqual(ret.y2, 6.4)
+
+        with self.assertRaises(TypeError):
+            line.move()
+
+        with self.assertRaises(TypeError):
+            line.move(1)
+
+        with self.assertRaises(TypeError):
+            line.move(1, 2, 3)
```

Comment:
Another test I could suggest adding is if the user does `line.move("3", "2")`

Reply:
this is a test that already exists in the rect_test.py in pygame as it handles rects, i dont think we need to test that but if @Starbuck5 thinks it is a good idea i would be more than happy to implement it